genai-perf/tests/test_rankings_converter.py

```diff
@@ -60,17 +76,17 @@ def test_convert_default(self):
                 {
                     "payload": [
                         {
-                            "query": {"text": "query 1"},
-                            "passages": [{"text": "passage 1"}, {"text": "passage 2"}],
+                            "query": "query 1",
+                            "passages": [{"text_input": "passage 1"}, {"text_input": "passage 2"}],
```

Comment:
Why the change from `text` to `text_input`? Is this a bug fix?

Reply:
I can undo this. We initially supported text for rankings/embeddings and text_input for everything else. We then decided to consolidate around text_input. Finally, we consolidated around text for all inputs, with backwards compatible support for text_input until we go GA.

I must have gotten mixed up while writing this test and changed which field we're using. It'll work, but there's no reason to use the deprecated field name here.

Reply:
Sounds like going forward we prefer/support `text` but this test is for backwards compatibility with `text_input`?

Reply:
That's correct.